src/fromager/commands/build.py

```diff
@@ -101,13 +101,20 @@ def build_sequence(
             source_download_url = entry["source_url"]
             req = Requirement(f"{dist_name}=={resolved_version}")
 
+            log_filename = str(resolved_version) + "-" + dist_name + ".log"
+            wheel_log = wkctx.work_dir / log_filename
```

Comment:
We should include the build tag in the filename. @shubhbapna do we have enough information to compute the build tag here before we build the wheel?

Reply:
I agree